input/definitions/MeasureReport/StructureDefinition-criteriaReference.json

```diff
@@ -0,0 +1,88 @@
+{
+  "resourceType": "StructureDefinition",
+  "id": "criteriaReference",
+  "url": "http://hl7.org/fhir/StructureDefinition/criteriaReference",
+  "version": "1.0.0",
+  "name": "CriteriaReferenceExtension",
+  "title": "Criteria Reference Extension",
+  "status": "active",
+  "experimental": false,
+  "date": "2023-12-15",
+  "publisher": "HL7 International / Clinical Decision Support",
+  "contact": [
+    {
+      "telecom": [
+        {
+          "system": "url",
+          "value": "http://www.hl7.org/Special/committees/dss"
+        }
+      ]
+    }
+  ],
+  "description": "Specifies which criteria is the input for calculations.",
```

Comment:
This description is incorrect, this extension is for identifying which criteria is responsible for the inclusion of an evaluated resource or supplemental data element in the measure report. Should be:
"Specifies which criteria a resource was considered for"

Reply:
```suggestion
  "description": "Specifies which criteria is the input for calculations. Specifies which criteria a resource was considered for.",
```